addons/website/static/src/snippets/s_facebook_page/options.js

```diff
@@ -93,25 +95,60 @@ options.registry.facebookPage = options.Class.extend({
         return this._markFbElement();
     },
 
+    /**
+     * Sets the Facebook page's height.
+     * @see this.selectClass for parameters
+     */
+    setHeight(previewMode, widgetValue, params) {
+        // Extract numeric value from widgetValue
+        const numericValue = parseFloat(widgetValue);
+        // Cases when widgetValue is invalid e.g. empty, not a number, or greater than 700
+        if (!numericValue || numericValue > 700) {
```

Comment:
In which case we will not get numeric value?
why not something like:
```
if (numericValue <= 700) {
    this.fbData.height = `${numericValue}px`;
}
this._getFbPageElement().style.height = this.fbData.height;
```

Reply:
Makes sense the parseFloat here will always return 0 even if empty string is passed or say any string, i tested it in console it returns NaN. Thanks for the catch.